Sources/Ink/Internal/List.swift

```diff
@@ -92,13 +125,26 @@ internal struct List: Fragment {
 
     func html(usingURLs urls: NamedURLCollection,
               modifiers: ModifierCollection) -> String {
-        let tagName = isOrdered ? "ol" : "ul"
+        let tagName: String
+        let startAttr: String
```

Comment:
Another nitpicky comment (that I'd be happy to fix in post in case you don't have time): In general, I'm not a big fan of abbreviations in variable names, since I think it makes them unnecessarily cryptic. So I'd recommend naming this `startAttribute`.